src/main/java/appeng/api/storage/cells/StorageCell.java

```diff
@@ -39,6 +39,12 @@ public interface StorageCell extends MEStorage {
      */
     double getIdleDrain();
 
+    /**
+     * Return whether the cell does not currently contain anything. Use this to determine when your cell should be
+     * storable in or able to store other kinds of cells without recurring.
+     */
+    boolean isEmpty();
```

Comment:
This method is badly named, it should be something like `canFitInsideCell`. This is also an API breaking change as-is.

Maybe we should use `canFitInsideContainerItems` instead, but that might not have a stack aware overload (yet).